products/BellHybrid/apps/application-bell-settings/models/DateTimeUnitsModel.cpp

```diff
@@ -62,6 +62,7 @@ namespace app::bell_settings
         timeSetListItem =
             new gui::TimeSetListItem(0U, 0U, 0, 0, utils::translate("app_bell_settings_time_units_time_message"));
         internalData.push_back(timeSetListItem);
+        timeSetListItem->getTimeFormat = [this](void) { return timeFmtSetListItem->getTimeFmt(); };
```

Comment:
You don't need to explicitly write `void` when defining functions taking no arguments